### PR TITLE
Add component toast

### PR DESCRIPTION
**Pull request by sah-shah7**
- Added Toast Component
-Updated Sidebar & TopBar links
-Updated Landing page anchor Tags
image

### CODE REVIEW

**Review by yashpatel1797 — APPROVED** [review]
Looks good to merge.

**Comment by sah-shah7** [comment]
PR received. Merging

File: Components/Avatars/avatar.html [hunk 1]
```diff
@@ -10,103 +10,103 @@
 </head>
 
 <body>
-  <!-- Navigation Top Bar Starts Here-->
-  <nav class="nav-container">
-    <div class="nav-bar-left">
+<!-- Navigation Top Bar Starts Here-->
+<nav class="nav-container">
+  <div class="nav-bar-left">
```
Comment (bishnoimukesh) [inline Components/Avatars/avatar.html:15]: don't change the format

File: Components/Avatars/avatar.html [hunk 2]
```diff
@@ -10,103 +10,103 @@
...
+          <li class="nav-bar-list-items nav-bar-list-items-text">
+              <a class="link-a link-a-font-color-black" href="https://github.com/sah-shah7/Sway-UI">Github</a>
+          </li>
       </ul>
-    </div>
-  </nav>
-  <!--Navigation top bar ends here-->
+  </div>
+</nav>
+<!--Navigation top bar ends here-->
```
Comment (bishnoimukesh) [inline Components/Avatars/avatar.html:36]: same

**Review by bishnoimukesh — COMMENTED** [review]
take care of document formatting
rest looks good to go

**Comment by prateek1361** [comment]
good to merge

**Pull request merged**